.gitignore

```diff
@@ -1,8 +1,132 @@
-# IDE-Files
-.idea
+# Byte-compiled / optimized / DLL files
```

Comment:
I do not get, why you delete my ignore-Stuff :D I think this is very personal? You ignore much stuff, which is not commonly part of the package. Maybe I missunderstood something?

Reply:
Nevermind. I just used the github template